contracts/lib/LibPerpetuals.sol

```diff
@@ -113,4 +114,16 @@ library Perpetuals {
             notFilled &&
             createdBefore;
     }
+
+    function getExecutionPrice(
+        Order calldata a,
+        Order calldata b
+    ) public pure returns (uint) {
+        bool aIsFirst = a.created <= b.created;
```

Comment:
...ternary?

Reply:
@sporejack ternary seems a bit overkill here no? A ternary where you go `isThingTrue ? true : false` is the same as returning `isThingTrue`